### DISCUSSION TITLE
Error: Could not find HHBlits database

### DISCUSSION THREAD

**Discussion by theabhirath** [KosinskiLab/AlphaPulldown#192] [General]
Hi, we are trying to use your package and are encountering an error when trying out an example. On running the command:

```
create_individual_features.py \
--fasta_paths=baits.fasta,example_1_sequences_shorter.fasta \
--data_dir=/home/balakumaran.c/alphafold_database/ \
--mgnify_database_path=/home/balakumaran.c/alphafold_database/mgnify/mgy_clusters_2018_12.fa \
--output_dir=/home/balakumaran.c/alphapulldown/output/ \
--save_msa_files=False \
--use_precomputed_msas=False \
--max_template_date="2022-03-30" \
--skip_existing=True \
```

We are encountering the following error: `ValueError: Could not find HHBlits database /home/balakumaran.c/alphafold_database/uniref30/UniRef30_2021_03`. The full error file is attached below.

Our database follows the format for the AlphaFold 2.3.0 database:

```
alphafold_database
|-- bfd
|-- mgnify
|-- params
|-- pdb70
|-- pdb_mmcif
|   `-- mmcif_files
|-- pdb_seqres
|-- small_bfd
|-- uniclust30
|   `-- uniclust30_2018_08
|-- uniprot
|-- uniref30
|   `-- UniRef30_2021_03
`-- uniref90
```

and so I am uncertain about what the error could be. Could you please help me with the same?

The code is being run on a cluster on a GPU node with Red Hat Enterprise (Kernel Linux 3.10.0-327.el7.x86_64). The GPU is a Tesla K40m.

Thanks,
Abhirath.

[output.txt](https://github.com/KosinskiLab/AlphaPulldown/files/13165223/output.txt)

**Comment by dingquanyu** [comment]
Hi 

Thanks for your interest in our package. There should be 7 files under the uniclust30 subfolder. Could you please double check the requirements by AlphaFold? Since this is irrelevant to the package itself, it will be moved to the discussion section.
